Dockerfile

```diff
@@ -24,20 +24,13 @@ EXPOSE 8888
 
 # install NodeJS 16.x (latest LTS until ~October 2022, https://nodejs.org/en/about/releases/)
 RUN \
-    curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - && \
-    sudo apt-get install -y nodejs
+    curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
+    apt-get install -y nodejs
 
 # install pyopenssl cryptography idna and requests is the same as installing
 # requests[security]
 RUN source activate base && \
-    conda install -c conda-forge ndg-httpsclient==0.5.1 \
-          pyasn1==0.4.5 \
-          pyopenssl==19.0.0 \
-          cryptography==2.7 \
-          idna==2.8 \
-          requests==2.21.0 \
-          beautifulsoup4==4.8.1 \
-          html5lib==1.0.1
+    conda update -n base -c defaults conda
```

Comment:
not strictly necessary since no more conda stuff is done in the Dockerfile, but I did use it whilst experimenting with conda and pip installation on the docker image.